xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1484,7 +1484,9 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
 
     stream->source = STREAM_SOURCE_DEMUX;
     stream->pPrivate = pStream;
-    stream->flags = (CDemuxStream::EFlags)pStream->disposition;
```

Comment:
flags are used all over the place, they are forwarded directly here: https://github.com/peak3d/xbmc/blob/1800fcf24736faf870da4e23ce28c3964019d2a5/xbmc/cores/VideoPlayer/VideoPlayer.cpp#L523

If we want to be safe, we would have to translate all used av flags to our flags